images/ipfailover/keepalived/lib/config-generators.sh

```diff
@@ -59,6 +59,11 @@ function generate_script_config() {
   if [ "$port" = "0" ]; then
     echo "   script \"true\""
   else
+    if [[ -n $HA_CHECK_SCRIPT ]]; then
+      echo "   if [[ -f $HA_CHECK_SCRIPT ]]; then"
```

Comment:
@stevekuznetsov This is creating a config file for keepalived. One of the parameters to keepalived is the name of a  check script that keepalived should call. The expansion must happen when the config file is created.